cotracker/README.md

```diff
@@ -0,0 +1,74 @@
+---
+Paper: https://arxiv.org/pdf/2307.07635.pdf
+Official Website: https://co-tracker.github.io/
+Demo: https://huggingface.co/spaces/facebook/cotracker
+---
+# CoTracker
+
+A new state of the art architecture for optical flow and particle tracking proposed by meta. The key innovation in this project is the use of Transformer Architecture and the idea that tracking multiple particles concurrently, can help boos overall accuracy.
```

Comment:
The key innovation in this project is the use of Transformer Architecture and the idea that tracking multiple particles concurrently, can help boos overall accuracy. -> This sentence is a bit weird maybe rephrase it...?